imagehash.py

```diff
@@ -390,10 +390,23 @@ def __eq__(self, other):
 			return False
 		return self.matches(other)
 
+	def __ne__(self, other):
+		if other is None:
+			return False
```

Comment:
should this be False?

Reply:
Hmm, good point. In which case, it's just `not self.matches(other)`
I was hoping adding __ne__() would fix the 3.5 and 2.7 errors I see on travis, but it seems not.